enclave/core/CMakeLists.txt

```diff
@@ -132,7 +132,10 @@ target_compile_options(oecore INTERFACE $<$<COMPILE_LANGUAGE:CXX>:-nostdinc++>)
 target_compile_definitions(oecore
     PUBLIC
     OE_BUILD_ENCLAVE
-    OE_API_VERSION=2)
+    # NOTE: This definition is public to the rest of our project's
```

Comment:
This is in #1513 but required to build this right now. Waiting for CI to start working again.